### PR TITLE
Allows bonesaw debraining

### PR DESCRIPTION
**Pull request by ArcLumin**
Title

### CODE REVIEW

**Comment by Kokojos** [comment]
yes +0.34

**Comment by JohnGinnane** [comment]
Can I suggest you put up a changelog so players who don't watch the repo know about this?

File: code/modules/surgery/organs/organ_external.dm [hunk 1]
```diff
@@ -240,7 +240,7 @@
 
 	add_fingerprint(user)
 
-	var/list/implements = list(/obj/item/weapon/circular_saw, /obj/item/weapon/melee/energy/sword/cyborg/saw, /obj/item/weapon/melee/arm_blade,\
+	var/list/implements = list(/obj/item/weapon/circular_saw, /obj/item/weapon/circular_saw/bonesaw, /obj/item/weapon/melee/energy/sword/cyborg/saw, /obj/item/weapon/melee/arm_blade,\
```
Comment (JohnGinnane) [inline code/modules/surgery/organs/organ_external.dm:243]: If I can be completely honest I'm not a fan of this list. Can we use any sharp object instead?

Comment (ArcLumin) [inline code/modules/surgery/organs/organ_external.dm:243]: Well, considering is_sharp includes lots of stuff that wouldn't be able to cut through bone, like glass shards, prolly not the best idea

**Comment by ArcLumin** [comment]
> Needing head input for brain output

Pure comedy

Busy right now, will make a cl later

File: code/modules/surgery/organs/organ_external.dm (hunk 1 above)
Comment (ArcLumin) [inline code/modules/surgery/organs/organ_external.dm:243]: Suppose you could add is_jagged and only allow jagged objects to delimb/saw bone, but eh.

Comment (JohnGinnane) [inline code/modules/surgery/organs/organ_external.dm:243]: We could expand on the sharp flag though, save some memory

**Comment by Carbonhell** [comment]
It'sabugfix, slip a changelog in and i'll merge

**Comment by JohnGinnane** [comment]
eh carbon it's very snowflakey with the list

**Comment by Carbonhell** [comment]
this is a bugfix tho, if you want to revamp how this is dealt with it goes in another pr,but yes i agree,we should make item modifiers like pointed and sharp, but more of em

**Comment by ArcLumin** [comment]
fuck changelogs

in all seriousness though when　I get back I'll make a  ｃｈａｎｇｅｌｏｇ

**Comment by ghost** [comment]
Yeah okay +1

**Comment by SplergeBucket** [comment]
+1

File: html/changelogs/ArcLumin - Bonesaw.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, this gets changed to [] after reading.  Just remove the brackets when you add neit.
+# Please surround your changes in  double quotes ("). It works without them, but if you use certain characters it screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  -  tweak: "Bonesaws can now debrain"
```
Comment (Carbonhell) [inline html/changelogs/ArcLumin - Bonesaw.yml]: only 1 space between - and tweak

Comment (ArcLumin) [inline html/changelogs/ArcLumin - Bonesaw.yml]: Hard to tell on my phone, thanks

**Pull request merged**